python/src/Distribution.i

```diff
@@ -199,6 +201,9 @@ class SciPyDistribution(PythonDistribution):
         moment = self._dist.moment(n)
         return [moment]
 
+    def isContinuous(self):
+        True
```

Comment:
return

this was probably untested

Reply:
Also, could you drop the line I temporarily added at 8b1a754 so the CI uses scipy 1.8 ?

Reply:
ok